gm4_apple_trees/data/gm4_fruiting_trees/functions/main.mcfunction

```diff
@@ -1,4 +1,8 @@
 # age fruiting leaves
 execute as @e[type=armor_stand,tag=gm4_fruiting_leaf] run function gm4_fruiting_trees:tree/leaf/fruiting/age
 
+# entity versioning
+execute as @e[type=armor_stand,tag=gm4_fruiting_leaf,tag=!smithed.entity] run function gm4_fruiting_trees:upgrade_path/smithed_compat
+execute as @e[type=marker,tag=gm4_fruiting_sapling,tag=!smithed.entity] run function gm4_fruiting_trees:upgrade_path/smithed_compat
```

Comment:
I'm not convinced these upgrade paths are worth it. And if they're done, what's the reason for using `gm4_entity_version` if you're not testing for that here in `main`? If you can check whether or not a version needs to be upgraded without this version score, I don't think the score is necessary.

Reply:
`gm4_entity_version` is for future use (if we need to change things, we currently need to check tags and then use a new tag which isn't sustainable). I could check for the score here, but that'd take an extra execute subcommand, which would be slightly less efficient, so I used a tag check wherever possible.